app/src/components/About.vue

```diff
@@ -79,12 +83,17 @@
 
 <script lang="ts">
 import { defineComponent, onMounted, ref } from 'vue';
+// --- Icons ---
 import { CloseIcon as XIcon } from '@fusion-icons/vue/interface';
 import { TwitterIcon as TwitterIcon } from '@fusion-icons/vue/interface';
 import { GithubIcon as GithubIcon } from '@fusion-icons/vue/interface';
 
+// package.json is used to inform the displayed version
 import packagejson from './../../package.json';
 
+// allow for the dgrants ID to be set in .env
+const dgrantsGrantID = import.meta.env.VITE_DGRANTS_GRANT_ID || false;
```

Comment:
If above comment is implemented, this becomes
```suggestion
const dgrantsGrantID = Boolean(import.meta.env.VITE_DGRANTS_GRANT_ID);
```

Reply:
this comment can be ignored, see above